sql/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -225,6 +224,39 @@ public void testErrorRecoveryFromErrorsOutsideSqlOperations() throws Exception {
         }
     }
 
+    @Test
+    public void testStatsTableSuccess() throws Exception {
+        try (Connection conn = DriverManager.getConnection(JDBC_POSTGRESQL_URL)) {
+            conn.setAutoCommit(true);
+            PreparedStatement preparedStatement = conn.prepareStatement("select name from sys.cluster");
+            preparedStatement.executeQuery();
+
+            preparedStatement = conn.prepareStatement("select stmt from sys.jobs_log order by ended desc;");
+            ResultSet resultSet = preparedStatement.executeQuery();
+            assertThat(resultSet.next(), is(true));
+            assertThat(resultSet.getString(1), is("select name from sys.cluster"));
+        }
+    }
+
+    @Test
+    public void testStatsTableFailure() throws Exception {
+        try (Connection conn = DriverManager.getConnection(JDBC_POSTGRESQL_URL)) {
+            conn.setAutoCommit(true);
+            PreparedStatement preparedStatement = conn.prepareStatement("create table t (a integer not null, b string)");
```

Comment:
You could do a `select invalid from sys.cluster` instead of create table + insert, would be faster.